core/client/queries/get-customer.ts

```diff
@@ -0,0 +1,23 @@
+import { getSessionCustomerAccessToken } from '~/auth';
+import { client } from '~/client';
+import { graphql, ResultOf } from '~/client/graphql';
+
+export const GetCustomerGroup = graphql(`
+  query CustomerGroup {
+    customer {
+      customerGroupId
+    }
+  }
+`);
+
+export type GetCustomerGroupResponse = ResultOf<typeof GetCustomerGroup>;
+
+export async function getCustomer() {
+  const customerAccessToken = await getSessionCustomerAccessToken();
+  const response = await client.fetch({
+    document: GetCustomerGroup,
+    customerAccessToken,
+  });
+
+  return { data: response.data };
```

Comment:
Not required, but you could destructure `{ data }` from the response directly then use shorthand object notation. Don't know if we have opinions on doing this.

```
const { data } = await client.fetch({
    document: GetCustomerGroup,
    customerAccessToken,
  });

  return { data };
```